pkg/server/auth/server.go

```diff
@@ -311,7 +334,7 @@ func (s *AuthServer) Callback() http.HandlerFunc {
 			return
 		}
 
-		token, err = s.oauth2Config(nil).Exchange(ctx, code)
+		token, err = s.oauth2Config(s.OIDCConfig.Scopes).Exchange(ctx, code)
```

Comment:
Is it important to include the scopes when doing the exchange?

Reply:
No, you're right, they're not, I think I did a quick `s/scopes/s.OIDCConfig.Scopes/` good catch :-)